include/swift/SIL/GraphOperationInfo.h

```diff
@@ -247,6 +254,13 @@ bool isShapeArrayPseudoAttr(StringRef attrName, SymbolicValue attrValue);
 int decodeShapeAttr(const ASTContext &ctx, SymbolicValue attr,
                     SmallVectorImpl<int64_t> &result);
 
+/// Return the TF_DataType represented by `value`.
+unsigned getTFDataType(SymbolicValue value);
+
+/// Return a SymbolicValue that represents the TF_DataType for the given swift
+/// type.
+SymbolicValue convertSwiftTypeToTFSymbolicValue(Type type);
```

Comment:
The name is a little unclear. If the return is always gonna be a const `TF_DataType`, I'd suggest calling it `convertTypeToConstantTFDataType`.